### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by cswimr**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* @galaxypedia-wiki/development-team
+
+content/wiki/ @galaxypedia-wiki/staff
```
[the remark this answers is not in the document]
Comment (cswimr) [reply .github/CODEOWNERS:3]: Believe you need to give them write permissions to this repository. I don't know if I can do that?

[the remark this answers is not in the document]
Comment (cswimr) [reply .github/CODEOWNERS:3]: NVM I can, should be fixed now

[the remark this answers is not in the document]
Comment (smallketchup82) [reply .github/CODEOWNERS:3]: they should have maintain

[the remark this answers is not in the document]
Comment (smallketchup82) [reply .github/CODEOWNERS:3]: kk

**Review by smallketchup82 — APPROVED** [review]

**Pull request merged**